packages/shared/src/modules/chat-message/libs/types/message-get-all-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type MessageResponseDto } from "./message-response-dto.type.js";
+
+type MessageGetAllResponseDto = {
+	messages: MessageResponseDto[];
```

Comment:
```suggestion
	items: MessageResponseDto[];
```
soon we will unify all `GetAll` types